Objects/unicodeobject.c

```diff
@@ -4700,6 +4700,8 @@ static Py_ssize_t
 ascii_decode(const char *start, const char *end, Py_UCS1 *dest)
 {
     const char *p = start;
+    const char *size_t_end  = end - SIZEOF_SIZE_T;
+    const char *unroll_end = end - (4 * SIZEOF_SIZE_T);
```

Comment:
I would rename to `unrolled_end` to make it consistent with the changes in find_max_char.h

Reply:
Done!